README.rst

```diff
@@ -193,6 +193,10 @@ For citations of included algorithms and sub-modules please see the references_.
 .. |mybinder| image:: https://mybinder.org/badge.svg
    :alt: My Binder
    :target: https://mybinder.org/v2/gh/MDAnalysis/binder-notebook/master
+   
+ .. |asv| image:: http://img.shields.io/badge/benchmarked%20by-asv-blue.svg?style=flat
```

Comment:
I think this successfully defines an ASV badge, but you haven't placed the badge anywhere visible! Line 7 in this file places the second row of badges, I think this could fit there? You can look at what the page looks like in your branch here: https://github.com/HenokB/mdanalysis/tree/patch-1

```suggestion
 .. |asv| image:: https://img.shields.io/badge/benchmarked%20by-asv-blue.svg?style=flat
```

Please also use HTTPS when possible :)

Reply:
Unfortunately you still haven't placed your badge anywhere. Have a look at line 7.

Reply:
> Unfortunately you still haven't placed your badge anywhere. Have a look at line 7.

Oh yeah, I didn't see it. I just added it now, thanks